marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/matching.sjs

```diff
@@ -65,14 +65,43 @@ function validateArtifact(artifact) {
 function defaultArtifact(artifactName) {
   const defaultPermissions = 'data-hub-common,read,data-hub-common,update';
   return {
-    artifactName,
-    collections: ['default-mastering'],
-    additionalCollections: [],
+    batchSize: 100,
+    threadCount: 1,
     sourceDatabase: dataHub.config.FINALDATABASE,
     targetDatabase: dataHub.config.FINALDATABASE,
-    provenanceGranularityLevel: 'coarse',
+    targetEntity: "Change this to a valid entity type name; e.g. Customer",
+    sourceQuery: "cts.collectionQuery('default-mapping')",
```

Comment:
I dont think we generate a curated document with this collection. Should this also be changed to read something like:
`"cts.collectionQuery('Change this to a valid collection name; e.g. Customer')"` ?

Reply:
As per the story, the values should be from defaultFlow.flow.json; nevertheless I think this can be changed. 
@ryanjdew ,
Can you please weigh in on this as well ?

Reply:
@bsrikan Is correct. We should adjust this as a value for the user to change.